docs/attributes-registry/system.md

```diff
@@ -0,0 +1,160 @@
+# System
+
+<!-- toc -->
+
+- [CPU attributes](#cpu-attributes)
+- [Memory attributes](#memory-attributes)
+- [Paging attributes](#paging-attributes)
+- [Filesystem attributes](#filesystem-attributes)
+- [Network attributes](#network-attributes)
+- [Process attributes](#process-attributes)
+
+<!-- tocstop -->
+
+<!-- semconv registry.system(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `system.device` | string | The device identifier | `(identifier)` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+## CPU attributes
+
+<!-- semconv registry.system.cpu(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  | Stability |
+|---|---|---|---|---|
+| `system.cpu.logical_number` | int | The logical CPU number [0..n-1] | `1` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `system.cpu.state` | string | The CPU state for this data point. A system's CPU SHOULD be characterized *either* by data points with no `state` labels, *or only* data points with `state` labels. | `idle`; `interrupt` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Hey @gregkalapos !

In the registry the attributes should be generic and not specifically tied to a signal. The `The CPU state for this data point`  part and specifically the `data point` is a metric concept.
We have already discussed about the `.cpu.state` attribute at https://github.com/open-telemetry/semantic-conventions/pull/681#discussion_r1476037740 and https://github.com/open-telemetry/semantic-conventions/pull/282#discussion_r1528298983.

I would suggest (for now) we make the description signal-generic here removing  the "`data point`" part and then override it within the `system` metrics definition (we did the same in this [`container` metrics PR](https://github.com/open-telemetry/semantic-conventions/pull/282/files#diff-84fd4755368124a302b57f80ac0085c6ec69fca40dc49f183b9602838cd7c956R13))

I have filed https://github.com/open-telemetry/semantic-conventions/issues/840 to discuss a unification of this attribute using a union set of the possible values of the 3 different metric namespaces.

Reply:
I think we can do that in two steps:

1. Move all the existing attributes (as is) to the registry (this PR)
2. Do the change afterwards to consolidate all the `*.cpu.state` attributes (separate PR)

@ChrsMark WDYT?

Reply:
Yeap, I agree:

> I would suggest (for now) we make the description signal-generic here removing the "data point" part and then override it within the system metrics definition (we did the same in this [container metrics PR](https://github.com/open-telemetry/semantic-conventions/pull/282/files#diff-84fd4755368124a302b57f80ac0085c6ec69fca40dc49f183b9602838cd7c956R13))

Reply:
Seems we had different conclusion on this :) First of all, I totally agree, attributes should be signal agnostic.

My understanding was that we say that in this PR we just move attributes to the registry as is - without changing anything.

And then in a follow-up PR we change them to be signal agnostic.  That was point 1 from @AlexanderWert.

So, just to make sure we are all on the same page: 
Question: Do we want to just move things to the registry in this PR as is without any change (option1) or do we want to also adapt things to make it signal agnostic (option2)?

I'm fine with both options - just wanna make sure we are all on the same page. Maybe it was just me misinterpreting.

Reply:
Ah ok I see the confusion. I think point `2` was about consolidating the attributes as it is proposed at https://github.com/open-telemetry/semantic-conventions/issues/840. Point `1` includes moving the attributes to the registry but in the correct way, which means making them signal agnostic.

Reply:
Ok, let's move on with that. Adapting the PR.